HelloWorker/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+pushd "../SpatialOS/scripts"
```

Comment:
This script will fail when called from a different folder. Make sure you use absolute filepaths or `cd "$(dirname "$0")` across the board.

Reply:
done